### DISCUSSION TITLE
New user help

### DISCUSSION THREAD

**Discussion by itsarham001** [community/community#42448] [New to GitHub]
First of all I don't know what should I do can anyone help ?

**Comment by jfullstackdev** [comment]
are you referring to Git and GitHub ? 

I have my answer here 

https://github.com/community/community/discussions/31642#discussioncomment-3536733

**Comment by DSDmark** [comment]
Hi @itsarham001,
   
> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

And also, why any chance you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc).  its a video. so, just ***sit back and relax***

**Reply by DSDmark** [reply]
> First of all I don't know what should I do can anyone help ?

Can you **specific more** about the details of what you want? For better understanding 😅. Also, you can add some **screenshots** or **error** which you were facing 👍.
